docs/www/nodule-grid.js

```diff
@@ -377,6 +377,368 @@
+                        "Lung-RADS assessment": "Lung-rads 2",
+                        "Lesion Review Status": "Accepted",
+                        "Maximum 2D diameter": "5.4",
+                        "Maximum 3D diameter": "5.4",
+                        "Maximum perpendicular 2D diameter": "4.0",
+                        "Mean 2D diameter": "4.7",
+                        "Volume": "21.4"
+                    }
+                ];
+
+                //VAPALS.displayNotification("Updating Data");
+                if (!confirm("Do you really want to do the data update?")) {
+                    return;
+                }
+
+                // if number of nodules shown is less than the number of entries in theData array then increase the number of nodules to they match
+                while (settings.getNoduleCount() < theData.length) {
+                    _addNodule()
+                }
+
+                noduleId = 1;
+                theData.forEach(obj => {
+                    Object.entries(obj).forEach(([key, value]) => {
+                        // Need to work: “Finding”: “Pulmonary nodule”,
+
+                        // Need to work: “Finding site”: “Upper lobe of right lung”
+                        //So assuming using the "Most likely location" field ( cect1ll ) of the form.
+                        //“Upper lobe of right lung” = "RUL"
+                        //"Upper lobe of left lung" = "LUL"
+                        //"Lower lobe of right lung" = "RLL"
+                        //"Lower lobe of left lung" = "LLL"
+                        if (key === "Finding site") {
+                            ll = "#cect" + noduleId + "ll";
+                            if (!$(ll).hasClass("update-data")) {
+                                $(ll).addClass("update-data");
+                                $(ll).attr("original-value", $(ll).val());
+                                $("<button class=revert-field title='Revert back to the original data for this field' onclick=revertField(this)><i class='fa fa-arrow-circle-left'></i></button>").insertAfter(ll);
+                                $(ll).parent().addClass("update-data-parent");
+                            }
+                            if (value == "Upper lobe of right lung") {
+                                $(ll).val("rul");
+                            } else if (value == "Upper lobe of left lung") {
+                                $(ll).val("lul");
+                            } else if (value == "Lower lobe of right lung") {
+                                $(ll).val("rll");
+                            } else if (value == "Lower lobe of left lung") {
+                                $(ll).val("lll");
+                            }
+                        }
+
+                        // “Attenuation Characteristic”: “Solid”, bold: Consistency - cect1nt
+                        // values include: PartSolid, Solid, Unknown
```

Comment:
Remove these comments. They are not useful because you can simply read the code and infer this information.